spec/mongoid/query_cache_spec.rb

```diff
@@ -255,24 +255,50 @@
     end
 
     it 'returns all documents' do
-      expect(Person.all.to_a.count).to eq(3)
-      Person.first
-      expect(Person.all.to_a.count).to eq(3)
+      # Mongoid adds a sort by _id to the Person.first call, which is why
+      # these commands issue two queries instead of one.
+      expect_query(2) do
```

Comment:
I added a note to MONGOID-4972 to document this.

Reply:
:+1: